### ISSUE TITLE
Screenshot with undesired menu box

### ISSUE THREAD

**Issue by juliohm** [c42f/displaz#91]
When we take a screenshot, this is how it is saved (PNG):

![lidar2](https://cloud.githubusercontent.com/assets/3345261/12699948/438b7c36-c784-11e5-8a34-de01a676b126.png)

**Comment by c42f** [comment]
Yes, this is definitely a bug, but fixing it properly wasn't easy last time I looked.

As a very easy workaround, you can press F9 directly without using the menu.  You may also turn off any unwanted decorations using the View menu.

**Comment by nigels-com** [comment]
Yeah, it's better to render offscreen for nice results with images.

**Comment by c42f** [comment]
@nigels-com - there's even supposedly a function in QGLWidget to do exactly that, but it forces a different QGLContext, which led to strange crashing behaviour last time I tried and I wasn't sufficiently motiviated to fix it.  May be better/easier in Qt5, I haven't looked.

**Comment by c42f** [comment]
Actually this is probably quite a lot easier to fix than last time I thought about it, with the points already rendered to an offscreen buffer internally.

Side note - it should also be possible to optionally render screenshots with antialiasing, which would make them a lot nicer in many cases such as this one.

**Comment by nigels-com** [comment]
FBOs might be nice for this.  For a stationary camera you could skip all the heavyweight rendering and do annotations, labels and so as a compositing-style step.

**Comment by c42f** [comment]
I guess that's kind of how it works already with the bounding boxes, cursor and axes drawn onto the backbuffer after blitting in the points which are rendered offscreen in multiple passes.

**Comment by juliohm** [comment]
What is the status of this issue?

**Comment by c42f** [comment]
It's still open, and I'm not sure when I'll get to it.  Alas I don't have a lot of free time right now.

Luckily, as stated above there's a very simple workaround: just use the keyboard shortcut.
